test/build/legacy-mount-scripts/legacy-mount.test.js

```diff
@@ -1,22 +1,33 @@
 const fs = require('fs');
 const path = require('path');
-const {setupBuildTest} = require('../../test-utils');
+const snowpack = require('../../../snowpack');
 
-const cwd = path.join(__dirname, 'build');
+const TEST_ROOT = __dirname;
+const TEST_OUT = path.join(__dirname, 'build');
 
 describe('config: mount scripts (legacy)', () => {
-  beforeAll(() => {
-    setupBuildTest(__dirname);
+  beforeAll(async () => {
```

Comment:
I'm actually not sure why this test is failing, but because this support will be going out in v3.0 and we're removing this legacy `mount` support in v3.0 anyway, I'm +1 for just removing this test of legacy behavior.

Reply:
Resolved in (73bdab6)[https://github.com/snowpackjs/snowpack/pull/1928/commits/73bdab6de514f3452138f66d5e781601af4e4beb]